code/modules/vehicle/vehicle.dm

```diff
@@ -146,8 +150,10 @@
 	if(has_gravity(src))
 		return 1
 
-	if(pulledby && pulledby != buckled_mob)	// no pulling the vehicle you're driving through space!
-		return 1
+	if(pulledby)
```

Comment:
Um, what is this doing?

Reply:
its the long way of doing the old check considering buckled_mob is now a list and we have to iterate through it. see lines below.